backend/rdp/output.c

```diff
@@ -0,0 +1,289 @@
+#include <assert.h>
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include <stdlib.h>
+#include <wayland-server.h>
+#include <wlr/types/wlr_output.h>
+#include <wlr/interfaces/wlr_output.h>
+#include <wlr/util/log.h>
+#include <wlr/render/wlr_renderer.h>
+#include "backend/rdp.h"
+#include "util/signal.h"
+
+static struct wlr_rdp_output *rdp_output_from_output(
+		struct wlr_output *wlr_output) {
+	assert(wlr_output_is_rdp(wlr_output));
+	return (struct wlr_rdp_output *)wlr_output;
+}
+
+static EGLSurface egl_create_surface(struct wlr_egl *egl, unsigned int width,
+		unsigned int height) {
+	EGLint attribs[] = {
+		EGL_WIDTH, width,
+		EGL_HEIGHT, height,
+		EGL_RED_SIZE, 1,
+		EGL_GREEN_SIZE, 1,
+		EGL_BLUE_SIZE, 1,
```

Comment:
Actually, we already ask for red/green/blue in `backend.c`, and these are invalid for `eglCreatePbufferSurface`. Can you revert this change?